apps/wallet/src/background/keyring/VaultStorage.ts

```diff
@@ -150,7 +150,7 @@ class VaultStorageClass {
         this.#vault.importedKeypairs.push(keypairToImport);
         await setToLocalStorage(VAULT_KEY, await this.#vault.encrypt(password));
         await this.updateSessionStorage();
-        return true;
```

Comment:
we could return `null` above instead of `false` in the case of `isDuplicate`

Reply:
done